pacbot_rs/src/game_state/env.rs

```diff
@@ -340,14 +355,16 @@ impl PacmanGym {
             } else {
                 let state_index =
                     if self.game_state.state == GameStateState::Chase { 1 } else { 0 };
-                state[(state_index, pos.0, pos.1)] = 1.0;
```

Comment:
One thought I have here is that this makes "near the end of chase", "near the end of scatter", and "near the end of frightened" very similar to each other in obs space, when we may want those to be more orthogonal.

Maybe we could have a duration channel plus three one-hot channels? Thoughts?

Reply:
Thinking this is *probably* okay for now, the network *should* be able to pick up on the distinction. An easy way to make them farther apart without adding another channel could be to set the non-active state channels to -1.